src/components/AvatarWithDisplayName.js

```diff
@@ -45,7 +45,7 @@ const defaultProps = {
 
 const AvatarWithDisplayName = (props) => {
     const title = ReportUtils.getDisplayNameForParticipant(props.report.ownerEmail, true);
-    const subtitle = ReportUtils.getChatRoomSubtitle(props.report, props.policies);
```

Comment:
This second param wasn't doing anything because the method only had one parameter. Now that I added a second, this could cause problems, so I removed it

Reply:
> This second param wasn't doing anything because the method only had one parameter. Now that I added a second, this could cause problems, so I removed it

added `second` means second param or `sec`? so `getChatRoomSubtitle` method doesn't have the second param, right?

Reply:
I added another parameter to the method, and this call of the method was already sending a second parameter but it wasn't doing anything. I had to remove it because otherwise the code would think the `props.policies` is a parentReport when it isn't